.travis.yml

```diff
@@ -5,4 +5,18 @@ before_cache:
 cache:
   directories:
     - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
\ No newline at end of file
+    - $HOME/.gradle/wrapper/
+deploy:
+  on:
+    branch: master
+  provider: elasticbeanstalk
+  access_key_id:
+    $AWS_ACCESS_KEY_ID
+  secret_access_key:
+    $AWS_SECRET_ACCESS_KEY
```

Comment:
*question* What kind of access does this user have? Is it scoped to only update the elasticbeanstalk instance?